LibrarySystemBackend/LibrarySystem/LibrarySystem/Services/ResevationService.cs

```diff
@@ -24,15 +24,17 @@ public async Task<bool> AddResevationAsync(ResevationViewModel resevationViewMod
         {
             try
             {
-                var resevations = new Resevation
+                var resevations = new Reservation
                 {
                     ReservationDate =new DateTime( resevationViewModel.ReservationDate.Ticks,DateTimeKind.Utc),
                     Status = resevationViewModel.Status,
                     MemberID = resevationViewModel.MemberID,
                     StaffID = resevationViewModel.StaffID,
-                    BookID = resevationViewModel.BookID
+                    BookID = resevationViewModel.BookID,
```

Comment:
Do not need to add Id to Add reservation method (because it is an auto-generated value)